websites/models.py

```diff
@@ -57,6 +57,7 @@ class Website(TimestampedModel):
         "WebsiteStarter", null=True, blank=True, on_delete=models.CASCADE
     )
     name = models.CharField(max_length=512, db_index=True, unique=True)
+    short_id = models.CharField(max_length=100, db_index=True, unique=True, null=False)
```

Comment:
No need to change but FYI `db_index=True` is redundant here, `unique=True` will add an index for its unique constraint